eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -213,8 +235,10 @@ public async Task<bool> CheckTestDiscoveryAsync()
         public async Task<int> RunTestsAsync()
         {
             var exitCode = 0;
-            try 
+            try
             {
+                // Helix timeout is 30 minutes right now
+                var cts = new CancellationTokenSource(TimeSpan.FromMinutes(25));
```

Comment:
nit: This is configurable from our side isn't it?  should we pass this in as an option?

Reply:
Yes

Reply:
Also this seems too long no?

Reply:
> Also this seems too long no?

I saw IIS test runs taking 19 minutes